winner-tool/src/main.rs

```diff
@@ -136,12 +124,12 @@ fn main() {
         override_num_threads: Some(1),
     };
 
-    let excluded_set = {
-        let mut set = HashSet::new();
-        set.insert(baseline_id);
-        set.insert(bootstrap_id);
-        set
-    };
+    let excluded_set = exclude_pubkeys
```

Comment:
I think you can write this in one line as:
`let excluded_set: HashSet<Pubkey> = exclude_pubkeys.into_iter().collect();`
or
`let excluded_set: HashSet<Pubkey> = HashSet::from_iter(exclude_pubkeys);`
(will have to `use std::iter::FromIterator;` for the latter)

Reply:
Much better, thanks!